blocks/library/gallery/gallery-image.js

```diff
@@ -75,10 +76,10 @@ class GalleryImage extends Component {
 	}
 
 	componentWillReceiveProps( { isSelected, image } ) {
-		if ( image && image.data && ! this.props.url ) {
+		if ( image && ! this.props.url ) {
```

Comment:
When will `image` be falsey?

Reply:
Before the API loads the image?

Reply:
Oh I didn't see the object fallback

Reply:
Not related to this pull request, but why do we access `this.props.url` here? Should this not be from the `nextProps` argument?